### ISSUE TITLE
Default labels on all Xray test execution issues

### ISSUE THREAD

**Issue by aufdenpunkt** [Qytera-Gmbh/cypress-xray-plugin#349]
Hi ,

First of all I would like to say thanks for this great plugin! I really like the simplicity of the configuration and that it works out of the box.

I was wondering if it is possible to add labels to the created test execution issues in Xray. The case is that we have some team specific labels in Jira that we need to add to each issue.

Best and thanks,
Chris

**Comment by csvtuda** [comment]
Hi @aufdenpunkt,

thanks for the feature request and sorry for the late reply. I tested if Xray's API supports setting labels, and both server and cloud APIs actually do indeed support it (arbitrary Jira fields actually).

I will hopefully include the new configuration in the next minor release.

**Comment by aufdenpunkt** [comment]
Hi @csvtuda,

Thank you very much for taking care! So, our team doesn't need dynamic values, but from my perspective, it is indeed a valid use case to use different labels for different tests, statuses, and so on. 🤔

Looking forward to the next release! 🙏

**Comment by csvtuda** [comment]
Hey @aufdenpunkt,

I have just released version [7.2.0](https://www.npmjs.com/package/cypress-xray-plugin/v/7.2.0) ([changelog](https://github.com/Qytera-Gmbh/cypress-xray-plugin/blob/main/CHANGELOG.md#720)). You should now be able to set test execution issue labels as follows:

```js
await configureXrayPlugin(on, config, {
    jira: {
        testExecutionIssue: {
            fields: {
                labels: ['x', 'y', 'z']
            }
        }
    },
});
```

**Comment by aufdenpunkt** [comment]
Hi @csvtuda,

amazing, this is exactly what we need! Thank you verymuch! 🙏
